### PR TITLE
26th june 2022

### PR DESCRIPTION
**Pull request by springcoil**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-A super organization is loosely modelled on superuser. So you can as a company ACME have specific criteria and permissions - and then you can share these with your child organisations. This allows you fine grained control of users and their roles and permissions and the ability to share settings and voices across orgs. We were informed by IAM from AWS in our design. If you want access let us know, we're working hard on this. This is part of a whole 
-
-- *Voices* We have some great partnerships coming up with 2 new voice providers. Reach out to us if you want to know more :) 
-## Coming soon 
-Under msnr we will soon have another german voice. We're testing this with some beta customers the `margareta-v1` voice. 
-
+- Friday 26th June 2022
+v.0.16.3 
+## SuperOrg
+- Listing superorgs - the abiltiy to list these organisations. `list_org_id`
```
Comment (zeritte) [inline CHANGELOG.md:5]: id remove `list_org_id` as wearent sure what to call it atm

Comment (springcoil) [reply CHANGELOG.md]: What are we calling them?

**Comment by gromoglasov** [comment]
Where are the webhooks docs?:)

File: CHANGELOG.md (hunk 1 above)
Comment (zeritte) [reply CHANGELOG.md]: its not added to sdks yet, i should create a ticket for @guillem-211

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,35 +1,63 @@
...
-  v0.16.2
-
-## Enhancements
-
-- We introduced a new function called `set_assume_org_id` for incoming super organization feature. By using this method, you can assume the id of a child organization as their super organization, and make your calls on behalf of them.
-
-A super organization is loosely modelled on superuser. So you can as a company ACME have specific criteria and permissions - and then you can share these with your child organisations. This allows you fine grained control of users and their roles and permissions and the ability to share settings and voices across orgs. We were informed by IAM from AWS in our design. If you want access let us know, we're working hard on this. This is part of a whole 
+```
+python 
+apiaudio.Mastering().create(scriptId=script_id, callback_url='call-based callback url')
```
Comment (Sjhunt93) [inline CHANGELOG.md:16]: Do we have an end-to-end example or this. What type of endpoint is required for a webhook? (POST, GET) etc?

Comment (Sjhunt93) [reply CHANGELOG.md:16]: also an example of what is sent from our end (i.e. json)

**Review by Sjhunt93 — APPROVED** [review]
Ive added comments, happy to approve but check them

File: CHANGELOG.md (hunk 2 above)
Comment (springcoil) [reply CHANGELOG.md:16]: @zeritte Do we?

Comment (zeritte) [reply CHANGELOG.md:16]: these will be in the user facing documentation. they are in the technical docs at the moment, if you wanna take a look. we make a post request for webhooks

Comment (springcoil) [reply CHANGELOG.md:16]: @Sjhunt93 I think showcasing that we have webhooks is enough for now we can update it in the future.

**Pull request merged**